src/components/InfoTip/Menu.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Dropdown } from '../Dropdown/index';
+
+class Menu extends React.Component {
+  render() {
+    const { children, ...props } = this.props;
+
+    return (
+      <Dropdown.Menu className="infotip bottom-right" {...props}>
```

Comment:
Can you also support adding different classes while consuming?

You can see an example here:
https://github.com/patternfly/patternfly-react/blob/master/src/components/Alert/Alert.js#L19-L21